ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool
+
+import scopt.{OParser, Read}
+
+object Cli {
+  def config(args: Array[String]): Option[Config] =
+    OParser.parse(parser, args, defaultConfig)
+
+  private val parser = {
+    val builder = OParser.builder[Config]
+    import builder._
+    import Reads._
+    OParser.sequence(
+      programName("ledger-api-bench-tool"),
+      head("A tool for measuring transaction streaming performance of a ledger."),
+      opt[String]("hostname")
+        .abbr("h")
+        .text("Ledger API hostname")
+        .optional()
+        .action((host, config) => config.copy(ledger = config.ledger.copy(hostname = host))),
+      opt[Int]("port")
+        .abbr("p")
+        .text("Ledger API port")
+        .optional()
+        .action((port, config) => config.copy(ledger = config.ledger.copy(port = port))),
+      opt[Config.StreamConfig]("consume-stream")
+        .abbr("s")
+        .text(
+          s"Stream configuration."
+        )
+        .valueName("streamType=<transactions|transactionTrees>,name=<streamName>,party=<party>")
```

Comment:
we tend to use dash separators in command-line options
```suggestion
        .valueName("streamType=<transactions|transaction-trees>,name=<streamName>,party=<party>")
```